src/sentry/api/endpoints/project_templates_index.py

```diff
@@ -60,3 +66,35 @@ def get(self, request: Request, organization: Organization) -> Response:
             on_results=lambda x: serialize(x, request.user, ProjectTemplateSerializer()),
             paginator_cls=OffsetPaginator,
         )
+
+    @ensure_rollout_enabled(PROJECT_TEMPLATE_FEATURE_FLAG)
+    def post(self, request: Request, organization: Organization) -> Response:
+        """
+        Create a new Project Template.
+
+        Create a new project template for the organization.
```

Comment:
Is it intentional to have this twice?

Reply:
NGL, ai wrote this. hahah i'll clean it up.